### DISCUSSION TITLE
Source Google Sheets: problem with Initiate OAuth

### DISCUSSION THREAD

**Discussion by Alik1604** [airbytehq/airbyte#28735] [Connector Questions]
I need to create Google Sheet connection for my users. As i understood I can use this for my task

https://api.airbyte.com/v1/sources/initiateOAuth

this is body of request

{
  "name": "Google Sheets",
  "redirectUrl": "https://cloud.airbyte.io/v1/api/oauth/callback",
  "workspaceId": "********"
}

but I received this error

{
  "type": "https://reference.airbyte.com/reference/errors",
  "title": "value-not-found",
  "status": 400,
  "detail": "Submitted value could not be found: Google Sheets"
}

Maybe I incorrectly understood name field. Can you provide please example value for this field

**Comment by marcosmarxm** [comment]
@Alik1604 are you trying to create a connector using Oauth on Airbyte Cloud? If that's the case did you try to follow instructions in https://reference.airbyte.com/reference/creating-oauth-sources

**Reply by Alik1604** [reply]
Yes, I try to create source from Google Sheet of user

**Reply by Alik1604** [reply]
I followe this tutorial 
https://reference.airbyte.com/reference/creating-oauth-sources

And try this playground

https://reference.airbyte.com/reference/initiateoauth

![image](https://github.com/airbytehq/airbyte/assets/31893948/a4fbbef6-a5d2-460c-9bcc-a323d941f4e5)

[a comment by an automated account is not shown]
